aiven/client/cli.py

```diff
@@ -5000,6 +5000,89 @@ def application_user__delete(self) -> None:
         self.client.delete_application_user(organization_id=self.args.organization_id, user_id=self.args.user_id)
         print(f"Application user {self.args.user_id} deleted successfully.")
 
+    @arg.json
+    @arg.force
+    @arg.organization_id
+    @arg("--resource-type", choices=["organization", "project"], help="Resource type", required=True)
+    @arg("--project-id", help="Project ID, required if resource_type is 'project'")
```

Comment:
Does it not make sen to leave this a bit more generic (and support `organization_unit`):
```suggestion
    @arg("--resource-type", choices=["organization", 'organization_unit", "project"], help="Resource type", required=True)
    @arg("--resource-id", help="Project ID, required if resource_type is not 'organization'")
```

as the value is already collapsed below as:
```python
resource_id = self.args.organization_id if self.args.resource_type == "organization" else self.args.project_id
```